Application/Dependency Injection/DependencyInjection.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Reflection;
+using MediatR;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Application.Dependency_Injection
```

Comment:
You can set namespace 'Microsoft.Extensions.DependencyInjection;
https://docs.microsoft.com/en-us/aspnet/core/fundamentals/dependency-injection?view=aspnetcore-5.0
Register groups of services with extension methods chapter